tests/sequence_labelling/preprocess_test.py

```diff
@@ -144,7 +144,7 @@ def test_serialize_to_json(self, tmp_path):
         preprocessor.fit(features_batch)
         word_preprocessor = WordPreprocessor(feature_preprocessor=preprocessor)
 
-        serialised_file_path = os.path.join(tmp_path, "serialised.json")
+        serialised_file_path = os.path.join(str(tmp_path), "serialised.json")
```

Comment:
Just FYI: From previous experience I found that `Path` is generally accepted to most core python functions in Python 3.6+. For Python 3.5 that wasn't the case. But nowadays we probably don't need to support Python 3.5 anymore. (And this is just a test). Just mentioning it in case you come across it on an old platform.

Reply:
Perhaps also worth mentioning, `Path` has a `joinpath` method, which is also available via the `/` operator. (Of course, unlike `os.path.join` they would return a `Path`)